core/src/types.ts

```diff
@@ -167,6 +167,10 @@ export declare interface SurplusAuctionActive extends SurplusAuctionBase {
     bidEndDate?: Date;
     earliestEndDate: Date;
     state: 'just-started' | 'have-bids' | 'ready-for-collection' | 'requires-restart';
+    fetchedAt: Date;
+    isRestarting: boolean;
```

Comment:
@valiafetisov for the swap auctions we had a system of the following types:

- AuctionInitialInfo
- Auction
- AuctionTransaction

How do we plan to handle this for these new auctions? With this PR we need to store more information we would normally try to exclude from the Initial Auction Data (isRestarting, fetchedAt and MarketPrices).

Do you have any kind of idea to better structure the types?

Reply:
> * `SurplusAuctionTransaction` would contain most other fields like `marketPrice` or `...TransactionFee`-like fields

What I take from this, to simplify it for @aomafarag (correct me if I am wrong):
- for isRestarting look at the logic from the other PR and copy it over (at least via props or whatever else is needed)
- fetchedAt can be moved to the SurplusAuctionBase
- A new type (extending `SurplusAuctionActive`) called `SurplusAuctionTransaction` can be created, with the new properties needed (for now just `marketUnitPrice` and `marketUnitPriceToUnitPriceRatio` to keep it simple, the rest can be added later on)

Reply:
@aomafarag, please check the following link for the types that @KirillDogadin-std and I have agreed on using for his projects and some other components I am working on:

https://github.com/sidestream-tech/unified-auctions-ui/issues/341#issuecomment-1181436064

Reply:
Merged types from main into this PR. Everything should be available now!